tutorcredentials/__about__.py

```diff
@@ -1,2 +1 @@
 __version__ = "16.0.3"
-
```

Comment:
why removing a newline? It's a python file.

Reply:
I know that! :) 
there were 2 blank lines at the end of this file. 
![Screenshot from 2023-11-23 21-01-40](https://github.com/overhangio/tutor-credentials/assets/49523567/9c5de4fa-3871-416c-9f31-4f14c2774e3d)

GitHub doesn't show the `1 blank line` at the end of the files and only warns you if there are not any.